core/server/data/sitemap/manager.js

```diff
@@ -72,138 +72,6 @@ _.extend(SiteMapManager.prototype, {
         return this[type].siteMapContent;
     },
 
-    pageAdded: function (page) {
-        if (!this.initialized) {
-            return;
-        }
-
-        if (page.get('status') !== 'published') {
-            return;
-        }
-
-        this.pages.addUrl(page.toJSON());
-    },
-
-    pageEdited: function (page) {
-        if (!this.initialized) {
-            return;
-        }
-
-        var pageData = page.toJSON(),
-            wasPublished = page.updated('status') === 'published',
-            isPublished = pageData.status === 'published';
-
-        // Published status hasn't changed and it's published
-        if (isPublished === wasPublished && isPublished) {
-            this.pages.updateUrl(pageData);
-        } else if (!isPublished && wasPublished) {
-            // Handle page going from published to draft
-            this.pageDeleted(page);
-        } else if (isPublished && !wasPublished) {
-            // ... and draft to published
-            this.pageAdded(page);
-        }
-    },
-
-    pageDeleted: function (page) {
-        if (!this.initialized) {
-            return;
-        }
-
-        this.pages.removeUrl(page.toJSON());
-    },
-
-    postAdded: function (post) {
-        if (!this.initialized) {
-            return;
-        }
-
-        if (post.get('status') !== 'published') {
-            return;
-        }
-
-        this.posts.addUrl(post.toJSON());
-    },
-
-    postEdited: function (post) {
-        if (!this.initialized) {
-            return;
-        }
-
-        var postData = post.toJSON(),
-            wasPublished = post.updated('status') === 'published',
-            isPublished = postData.status === 'published';
-
-        // Published status hasn't changed and it's published
-        if (isPublished === wasPublished && isPublished) {
```

Comment:
Did all this logic make it over okay?  I'm not really sure what it all means now, I just kind of blacked out and the unit test passed when I came to if I remember correctly.